src/style.css

```diff
@@ -0,0 +1,363 @@
+
+/*STYLES THE OVERAL LAYOUT*/
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+    position: relative;
+}
+
+#main-container {
+    margin: 0 5%;
+    height: 100vh;
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+}
+
+section {
+    padding: 10px;
+    display: grid;
+    grid-template-rows: 40px 1fr;
+}
+
+footer {
+    font-size: 10px;
+    height: fit-content;
+}
+
+
+
+/*RECURRENT STYLES*/
```

Comment:
Using classes for general attributes, big like 👍

Reply:
thanks!